R/data-objects.R

```diff
@@ -27,23 +26,30 @@
 #' # authentication
 #' iauth()
 #'
-#' # some data
-#' foo <- data.frame(x = c(1, 8, 9), y = c("x", "y", "z"))
+#' # creates a csv file of the iris dataset
```

Comment:
"a csv" -> "a local csv"

Reply:
For this (and following comments), I'll take care of this documentation once the PR is merged :)